charts/fluent-bit/templates/_pod.tpl

```diff
@@ -22,6 +22,7 @@ hostAliases:
 {{- if .Values.initContainers }}
 initContainers:
```

Comment:
There are two consecutives array:

- if both are defined, it's ok
- but might fail with one (or two) `[]` in the middle if at least one is empty

Reply:
I dropped `initContainersTpl` in favor `initContainers` (see <https://github.com/fluent/helm-charts/pull/160#discussion_r717931497>)